stablehlo/transforms/optimization/StablehloAggressiveSimplification.cpp

```diff
@@ -284,6 +284,37 @@ class ConcatenateOpFlatten : public SimplifyOpRewritePattern<ConcatenateOp> {
   }
 };
 
+/////////////////////////////////
+// CustomCallOp
+/////////////////////////////////
+
+struct FixMhloBackendConfigAttribute final
```

Comment:
How about something like: `CustomCallUnregisteredBackendConfigToFfi`

More consistent with the op-first naming, also would like to leave "mhlo" out of this. Also also we should be able to delete this pattern in the near futrue if we migrate JAX after